### PR TITLE
request to update with editmoduleui2

### PR DESCRIPTION
**Pull request by a0129998**
1. merged updated/stud html pages
2. written tests for editModule and editMounting

### CODE REVIEW

File: components/handlers/module_edit.py [hunk 1]
```diff
@@ -84,3 +84,4 @@ def POST(self):
 
             raise web.seeother("individualModuleInfo?code="+module_code+"&targetAY="+\
                                ay_sem.replace(' ', '+').replace('/', '%2F'))
+
```
Comment (nlzz22) [inline components/handlers/module_edit.py]: trailing newline here.

File: components/handlers/module_overview.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def GET(self):
         system_message = SESSION['editModMsg']
         SESSION['editModMsg'] = None
 
-        return RENDER.viewModule_updated(module_info, self.fixed_mounting_plan,
+        return RENDER.viewModule(module_info, self.fixed_mounting_plan,
                                          self.tenta_mounting_plan, number_of_student_planning,
                                          system_message)
```
Comment (nlzz22) [inline components/handlers/module_overview.py]: Line 96: trailing whitespace
Lines 108,109: wrong continued indentation
This page is not pylinted.

File: components/handlers/module_view_in_ay_sem.py [hunk 3]
```diff
@@ -92,7 +92,7 @@ def GET(self):
         system_message = SESSION['editMountMsg']
         SESSION['editMountMsg'] = None
 
-        return RENDER.individualModuleInfo_updated(module_info, is_future_ay,
+        return RENDER.individualModuleInfo(module_info, is_future_ay,
                                                    target_ay_sem, self.mounting_status,
                                                    self.quota, system_message)
```
Comment (nlzz22) [inline components/handlers/module_view_in_ay_sem.py]: This file is not linted.

File: components/handlers/tentative_module_mountings.py [hunk 4]
```diff
@@ -104,7 +104,7 @@ def GET(self):
         self.populate_module_code_and_name()
         self.populate_mounting_values(selected_ay)
 
-        return RENDER.moduleMountingTentative_updated(selected_ay, self.full_mounting_plan)
+        return RENDER.moduleMountingTentative(selected_ay, self.full_mounting_plan)
 
 
     def POST(self):
```
Comment (nlzz22) [inline components/handlers/tentative_module_mountings.py]: Line 99 has a trailing whitespace

File: templates/individualModuleInfo.html [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
 $ moduleMC = moduleInfo[3]
 
 $ overviewURL = '/viewModule'
-$ title_string = 'Module Info for ' + moduleCode + ' in ' + ay
+$ title_string = 'Module Info for ' + moduleCode + ' in ' + aySem
 
 $var title:$title_string
 
-<div class="container-fluid">
+<div clas="container-fluid">
```
Comment (nlzz22) [inline templates/individualModuleInfo.html]: misspelled "class" as "clas"

File: templates/viewModule.html [hunk 6]
```diff
@@ -19,13 +18,13 @@
 $var title:$title_string
 
 <!--bottom half module info-->
-<div class="container-fluid">
+<div clas="container-fluid">
```
Comment (nlzz22) [inline templates/viewModule.html]: misspelled "class" as "clas"

File: test/test_edit_module.py [hunk 7]
```diff
@@ -0,0 +1,116 @@
+
```
Comment (nlzz22) [inline test/test_edit_module.py]: This file is not linted, pls pylint it.

File: test/test_edit_module_mounting.py [hunk 8]
```diff
@@ -0,0 +1,86 @@
+
```
Comment (nlzz22) [inline test/test_edit_module_mounting.py]: This file is not linted, pls pylint it.

File: templates/mountingEdit.html [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+                                <option></option>
+                                <option></option>
+                            </select>
+                        </div>-->
+                        <div class="form-group">
+                            <lable for="preclusion">Preclusion</lable>
+                            <textarea class="form-control" id="preclusion" placeholder="e.g. CS1010"></textarea>
+                        </div>
+                        <div class="form-group">
+                            <lable for="prerequisit">Preclusion</lable>
```
Comment (nlzz22) [inline templates/mountingEdit.html]: This should be "Prerequisite" instead of "Preclusion"

Note: All your "label" are misspelled as "lable"

**Comment by a0129998** [comment]
fixed requested changes, please review.

**Comment by a0129998** [comment]
changed misspelling of  prerequisite

**Pull request merged**